clr-k8s-examples/create_stack.sh

```diff
@@ -20,104 +20,226 @@ $(
+        echo "Waiting for Prometheus CRDs"
+        sleep 2
+    done
+
+    #Expose the dashboards
+    #kubectl --namespace monitoring port-forward svc/prometheus-k8s 9090 &
+    #kubectl --namespace monitoring port-forward svc/grafana 3000 &
+    #kubectl --namespace monitoring port-forward svc/alertmanager-main 9093 &
+}
+
+function dashboard(){
+    DASHBOARD_VER=${1:-v2.0.0-beta2}
+    DASHBOARD_URL="https://github.com/kubernetes/dashboard.git"
+    DASHBOARD_DIR="2-dashboard"
+    get_repo "${DASHBOARD_URL}" "${DASHBOARD_DIR}/overlays/${DASHBOARD_VER}"
+    set_repo_version "${DASHBOARD_VER}" "${DASHBOARD_DIR}/overlays/${DASHBOARD_VER}/dashboard"
+    kubectl apply -k "${DASHBOARD_DIR}/overlays/${DASHBOARD_VER}"
+}
+
+function ingres() {
+    INGRES_VER=${1:-nginx-0.25.0}
+    INGRES_URL="https://github.com/kubernetes/ingress-nginx.git"
+    INGRES_DIR="5-ingres-lb"
+    get_repo "${INGRES_URL}" "${INGRES_DIR}/overlays/${INGRES_VER}"
+    set_repo_version "${INGRES_VER}" "${INGRES_DIR}/overlays/${INGRES_VER}/ingress-nginx"
+    kubectl apply -k "${INGRES_DIR}/overlays/${INGRES_VER}"
+}
+
+function efk(){
+    EFK_VER=${1:-v1.15.1}
+    EFK_URL="https://github.com/kubernetes/kubernetes.git"
+    EFK_DIR="3-efk"
+    get_repo "${EFK_URL}" "${EFK_DIR}/overlays/${EFK_VER}"
+    set_repo_version "${EFK_VER}" "${EFK_DIR}/overlays/${EFK_VER}/kubernetes"
+    kubectl apply -k "${EFK_DIR}/overlays/${EFK_VER}"
+
+}
+
+function metallb(){
+    METALLB_VER=${1:-v0.7.3}
+    METALLB_URL="https://github.com/danderson/metallb.git"
+    METALLB_DIR="6-metal-lb"
+    get_repo "${METALLB_URL}" "${METALLB_DIR}/overlays/${METALLB_VER}"
+    set_repo_version "${METALLB_VER}" "${METALLB_DIR}/overlays/${METALLB_VER}/metallb"
+    kubectl apply -k "${METALLB_DIR}/overlays/${METALLB_VER}"
 
-	#Expose the dashboards
-	#kubectl --namespace monitoring port-forward svc/prometheus-k8s 9090 &
-	#kubectl --namespace monitoring port-forward svc/grafana 3000 &
-	#kubectl --namespace monitoring port-forward svc/alertmanager-main 9093 &
 }
 
 function miscellaneous() {
-	kubectl apply -f 2-dashboard/
-	kubectl apply -f 3-efk/
-	#Create an ingress load balancer
-	kubectl apply -f 5-ingres-lb/
 
-	#Create a bare metal load balancer.
-	#kubectl apply -f 6-metal-lb/metallb.yaml
+    # dashboard
+    dashboard
 
-	#The config map should be properly modified to pick a range that can live
-	#on this subnet behind the same gateway (i.e. same L2 domain)
-	#kubectl apply -f 6-metal-lb/example-layer2-config.yaml
+    # EFK
+    efk
+
+    #Create an ingress load balancer
+    ingres
+
+    #Create a bare metal load balancer.
+    #kubectl apply -f 6-metal-lb/metallb.yaml
+
+    #The config map should be properly modified to pick a range that can live
+    #on this subnet behind the same gateway (i.e. same L2 domain)
+    #kubectl apply -f 6-metal-lb/example-layer2-config.yaml
 }
 
 function minimal() {
-	cluster_init
```

Comment:
use shfmt https://github.com/mvdan/sh

Reply:
I formatted the file with the `sh` tool as prescribed.